services/QuillLMS/app/assets/stylesheets/shared/tabs.scss

```diff
@@ -92,6 +92,13 @@
           &.active {
             background: $quill-white;
             color: $quill-green;
+            text-decoration: none;
+          }
+
+          &:hover {
+            background-color: rgba(255, 255, 255, 0.5);
```

Comment:
Is it worth asking Jack if this corresponds to a color palette color?

Reply:
thanks for flagging this, he's going to add them to the palette